index.html

```diff
@@ -8,10 +8,30 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <nav>
-        <ul>
-            <li>Logo</li>
-        </ul>
-    </nav>
+    <div id="header-background">
+        <nav id="flex-nav">
+            <ul>
+                <li id="logo"><a href="index.html">My Logo</a></li>
+                <li id="burguer-menu-icon ">
+                    <img src="img/menu.svg" alt="burguer-menu" class="burguer-menu-vector">
+                </li>
+            </ul>
+        </nav>
+        <div id="primary-text">
+            <h2 id="primary-text-header">I'm Enoque <br/> Glad to see<br/>you!</h2>
+            <p id="hello-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p id="connect">LET'S CONNECT</p>
+            <div id="social">
+                <img src="./img/twitter.png" alt="twitter link">
+                <img src="./img/Linkedin.png" alt="Linkedin link">
+                <img src="./img/m.png" alt="m link">
+                <img src="./img/github.png" alt="twitter link">
+                <img src="./img/m2.png" alt="m2 link">
+            </div>
```

Comment:
Kindly use `ul` tag to group the icons, and `li` tag to list them with each icon having a link `a` tag. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons using the `href`
`<ul> `
`<li> <a href = 'icon-link'> put icon here </a> </li>`
`<li> <a href = 'icon-link'> put icon here </a> </li>`
`</ul>`